packages/jmix-react-core/src/app/Router.tsx

```diff
@@ -28,24 +29,47 @@ function exec(re: RegExp, str: string, keys: pathToRegexp.Key[]) {
   return result;
 }
 
+type TitleFormatter = (value: string) => string
+
+let titleFormatter: TitleFormatter | null = null
+export const registerTitleFormatter = (formatter: TitleFormatter) => {
+  titleFormatter = formatter
+}
+export const clearTitleFormatter = () => {
+  titleFormatter = null
+}
```

Comment:
I think we agreed to simplify this? I think we don't need to register title formatter function. If we need to do something before setting `document.title` (like i18n) we should do it before calling `setWindowTitle`.

Reply:
Sorry, I don't quite understand your idea.

Reply:
After a closer look I see that `RouteItem` store i18n key rather than a message itself, so routing needs to be aware of i18n... That doesn't seem right, but it will require substantial effort to fix, besides, this won't be the case in the new Screen API. So let's leave it as it is for now.